alacritty_terminal/src/event_loop.rs

```diff
@@ -158,28 +66,28 @@ where
         hold: bool,
         ref_test: bool,
     ) -> EventLoop<T, U> {
-        let (tx, rx) = channel::channel();
+        let (tx, rx) = mpsc::channel();
         EventLoop {
-            poll: mio::Poll::new().expect("create mio Poll"),
+            poll: polling::Poller::new().expect("create Poll").into(),
             pty,
             tx,
-            rx,
+            rx: PeekableReceiver::new(rx),
             terminal,
             event_proxy,
             hold,
             ref_test,
         }
     }
 
-    pub fn channel(&self) -> Sender<Msg> {
-        self.tx.clone()
+    pub fn channel(&self) -> EventLoopSender {
```

Comment:
Might not be a bad idea to rename this function to be a bit more clear.

Maybe `send_channel`?

Reply:
Not a bad suggestion, but won't block on this since it's unrelated to this PR.